src/ert/_c_wrappers/enkf/analysis_config.py

```diff
@@ -101,6 +101,9 @@ def from_dict(cls, config_dict) -> "AnalysisConfig":
         if min_realization == 0:
             min_realization = num_realization
         min_realization = min(min_realization, num_realization)
+        max_runtime = config_dict.get(ConfigKeys.MAX_RUNTIME, 0)
```

Comment:
Could you perhaps add an issue, referencing these lines saying that we need to solve this in a better way.